tests/console/yast2_scc.pm

```diff
@@ -33,9 +33,10 @@ sub run {
 
 sub post_fail_hook {
     my ($self) = shift;
+    $self->SUPER::post_fail_hook;
+    select_console 'log-console';
```

Comment:
We already switch to `log-console` in the parent postfail hook

Reply:
yes, you're right, that was not need it. Updated VR.